landcover.mss

```diff
@@ -845,21 +845,44 @@
     line-color: #444;
   }
   [feature = 'barrier_hedge'][zoom >= 16] {
-    line-width: 3;
-    line-color: #aed1a0;
+    line-width: 1.5;
+    line-color: @forest;
```

Comment:
Perhaps make a new color variable for that which is an alias for forest.
This makes customizetion easier

Reply:
That's a reasonable idea. I've pushed a new commit with this change.